src/components/common/HangoutBox/HangoutBox.css.ts

```diff
@@ -0,0 +1,64 @@
+import { COLORS, TEXT_STYLES } from '@/constants/styles';
+import { style } from '@vanilla-extract/css';
+
+export const boxWrap = style({
+  padding: 16,
+  background: COLORS.grayscale.white,
+  borderRadius: 12,
+  position: 'relative',
+});
+
+export const nonAcceptedBoxWrap = style([
+  boxWrap,
+  {
+    opacity: 0.55,
+    border: `1px dashed ${COLORS.grayscale.black}`,
+  },
+]);
+
+export const boxShadow = style({
+  boxShadow: `0px 0px 3px 2px rgba(46, 46, 46, 0.02)`,
+});
+
+export const mainInfo = style({
+  display: 'flex',
+  marginBottom: 12,
+});
+
+export const ddayBox = style({
+  width: 'fit-content',
+  padding: '5px 10px',
+  color: COLORS.grayscale.gray700,
+  backgroundColor: COLORS.grayscale.gray200,
+  borderRadius: 12,
+  marginRight: 12,
+  ...TEXT_STYLES.caption1,
+});
+
+export const name = style({
+  color: COLORS.grayscale.black,
+  ...TEXT_STYLES.body1B,
+});
```

Comment:
🥦 어엄청 사소한 거긴 한데, 보통 기존 텍스트 스타일을 여기서 오버라이딩하는 경우가 많을 테니 이렇게 순서를 바꾸면 조금 더 안전할 것 같아요!
```suggestion
export const name = style({
  ...TEXT_STYLES.body1B,
  color: COLORS.grayscale.black,
});
```